packages/app-bridge/src/utilities/color.spec.ts

```diff
@@ -0,0 +1,59 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { describe, expect, it } from 'vitest';
+import { rgbObjectToRgbString, rgbStringToRgbObject } from './color';
+
+describe('Color', () => {
+    describe('rgbStringToRgbObject', () => {
+        it.each([
+            { input: 'rgb(1,2,3)', expected: { red: 1, green: 2, blue: 3, alpha: 1 } },
+            { input: 'rgb(1, 2, 3)', expected: { red: 1, green: 2, blue: 3, alpha: 1 } },
+            { input: 'rgb( 1,  2,  3 )', expected: { red: 1, green: 2, blue: 3, alpha: 1 } },
+            { input: 'rgba(1,2,3,0.6)', expected: { red: 1, green: 2, blue: 3, alpha: 0.6 } },
+            { input: 'rgba(1, 2, 3, 0.6)', expected: { red: 1, green: 2, blue: 3, alpha: 0.6 } },
+            { input: 'rgba( 1    ,  2  ,  3 , 0.6 )', expected: { red: 1, green: 2, blue: 3, alpha: 0.6 } },
+            { input: '    rgba( 1    ,  2  ,  3 , 0.6 )    ', expected: { red: 1, green: 2, blue: 3, alpha: 0.6 } },
+            { input: 'rgb(1, 2, 3);    ', expected: { red: 1, green: 2, blue: 3, alpha: 1 } },
+            { input: 'rgba(1, 2, 3, 0.6);    ', expected: { red: 1, green: 2, blue: 3, alpha: 0.6 } },
+            { input: 'rgba(,,, 0.6);    ', expected: { red: 0, green: 0, blue: 0, alpha: 0.6 } },
```

Comment:
Maybe this and decimal values should throw an error too, wdyt? Currently decimal numbers seem to be converted, but are actually not valid in a rgba object.

Reply:
Fixed